backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -48,7 +44,5 @@ trait BackendLifecycleActorFactory {
   lazy val fileHashingFunction: Option[FileHashingFunction] = None
   lazy val fileHashingWorkerCount: Int = 50
 
-  final lazy val fileContentsHasherActor: ActorRef = actorSystem.actorOf(RoundRobinPool(fileHashingWorkerCount)
-    .props(FileHasherWorkerActor.props(fileHashingFunction)),
-    "FileContentsHasherActor")
+  def fileContentsHasherActorProps: Props = FileHasherWorkerActor.props(fileHashingFunction)
```

Comment:
It's definitely related, imho. I'll wire in a singleton cache `Map[Factory, Pool]` into the engine side for the moment. But yeah, these pools-of-actors-per-factory needs "love".